scale/port/schema.py

```diff
@@ -17,9 +17,9 @@ def __init__(self, key, details):
         '''Constructor sets basic attributes.
 
         :param key: A unique identifier clients can use to recognize the warning.
-        :type key: str
+        :type key: string
```

Comment:
Well my understanding from the Sphinx docs is that either works, but string seems to be preferred by PyCharm.